datasets/gbif.yaml

```diff
@@ -15,7 +15,8 @@ Tags:
   - bioinformatics
   - conservation
   - life sciences
-License: "This dataset is available under a [CC-BY](https://creativecommons.org/licenses/by/4.0/) license and with the GBIF [terms of use](https://www.gbif.org/terms). Please refer to the GBIF [citation guidelines](https://www.gbif.org/citation-guidelines) when using this dataset."
+License: "This dataset is available under a [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) license and with the GBIF [terms of use](https://www.gbif.org/terms). Please refer to the GBIF [citation guidelines](https://www.gbif.org/citation-guidelines) when using this dataset and include the relevant DOI found from [the table](https://www.gbif.org/occurrence-snapshots)."
```

Comment:
Suggest shortening License field to:

"[CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) and under the GBIF [terms of use](https://www.gbif.org/terms)"